Testscripts/Linux/utils.sh

```diff
@@ -2464,14 +2464,35 @@ function install_iperf3 () {
 			# iperf3 is not available in the repository of SLES 12
 			which iperf3
 			if [ $? -ne 0 ]; then
-				LogMsg "iperf3 is not installed. So, Installing iperf3 using rpm"
-				iperf_url="$PACKAGE_BLOB_LOCATION/iperf-sles-x86_64.rpm"
-				libiperf_url="$PACKAGE_BLOB_LOCATION/libiperf0-sles-x86_64.rpm"
-				rpm -ivh $iperf_url $libiperf_url
-				which iperf3
+				iperf3_version=3.2
+				iperf3_url=https://github.com/esnet/iperf/archive/$iperf3_version.tar.gz
+				update_repos
+				gcc -v
```

Comment:
Use ```command``` to check the existence of utility.

Reply:
This is one way around of the original way.